unity-renderer/Assets/DCLPlugins/ECS7/Systems/ScenesUiSystem/Tests/ECSScenesUiSystemShould.cs

```diff
@@ -1,6 +1,7 @@
 using DCL;
 using DCL.Controllers;
 using DCL.CRDT;
+using DCL.ECS7;
```

Comment:
??

Reply:
that one's needed for using `ComponentID.UI_CANVAS_INFORMATION`

Reply:
oh I was looking at the wrong file, I'll remove that one

Reply:
Done